ui/src/app/views/breadcrumb/component.spec.ts

```diff
@@ -131,4 +170,26 @@ describe('BreadcrumbComponent', () => {
       'Breadcrumb label should be "Workspace Name"');
   }));
 
+  it('should show intermediateBreadcrumbs when they have children', fakeAsync( () => {
+    router.navigate(['intermediateParent', 'intermediateChild', 'intermediateGrandchild']);
+    tick();
+    expect(testComponent.breadcrumbs.length).toBe(3);
+    expect(testComponent.breadcrumbs[0].label).toBe('intermediateParent');
+    expect(testComponent.breadcrumbs[1].label).toBe('intermediateChild');
+    expect(testComponent.breadcrumbs[2].label).toBe('intermediateGrandchild');
```

Comment:
I think this should just be "grandchild" for this test to make sense (looks like it is indeed a normal `breadcrumb`, not an `intermediateBreadcrumb`)